.github/ISSUE_TEMPLATE/issue_template.yml

```diff
@@ -0,0 +1,71 @@
+name: Cadastro de Palestra
+description: Use este template para cadastrar sua palestra.
+title: "[PALESTRA] "
+labels: ["palestra"]
+assignees: []
+
+body:
+  - type: input
+    id: titulo
+    attributes:
+      label: Título
+      description: Título descritivo e interessante.
+      placeholder: "Título da palestra"
+      #required: true
+
+  - type: input
+    id: palavras_chave
+    attributes:
+      label: Palavras-chave
+      description: Palavras-chave relacionadas à palestra.
+      placeholder: "keyword1, keyword2, keyword3"
+      #required: true
+
+  - type: input
+    id: nivel
+    attributes:
+      label: Nível
+      description: Nível da palestra.
+      placeholder: "básico"
+      #required: true
+
+  - type: input
+    id: palestrante
+    attributes:
+      label: Palestrante
+      description: Nome do palestrante com link para a issue criada com seus dados.
+      placeholder: "[Nome Completo](link da issue)"
+      #required: true
+
+  - type: textarea
+    id: descricao
+    attributes:
+      label: Descrição da palestra
+      description: Uma descrição detalhada da palestra.
+      placeholder: "Lorem ipsum dolor sit amet, consectetur adipisicing elit..."
+      #required: true
+
+  - type: input
+    id: slides
+    attributes:
+      label: Slides
+      description: Link para os slides publicados (opcional).
+      placeholder: "link para os slides"
+      #required: false
```

Comment:
O default é true ou false?

Reply:
Para alguns campos como name, description e body o default é true. Para o restante costuma ser false, mas as preferi optar por deixar a estrurura de validação forçada para ajustar manualmente depois se necessário.